packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -412,3 +407,36 @@ class _RefreshOnGCToggleButton extends StatelessWidget {
     );
   }
 }
+
+class _ProfileHelpLink extends StatelessWidget {
+  const _ProfileHelpLink({Key? key}) : super(key: key);
+
+  static const _documentationTopic = 'profile';
+
+  @override
+  Widget build(BuildContext context) {
+    return HelpButtonWithDialog(
+      gaScreen: analytics_constants.memory,
+      gaSelection:
+          analytics_constants.topicDocumentationButton(_documentationTopic),
+      dialogTitle: 'Memory Allocation Profile Help',
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.end,
```

Comment:
this seems odd. Why would we need to align to the end of the column? If anything, I'd think we would use start

Reply:
The text itself is aligned to the left. Don't we want link 'more info' to be on the right?

Reply:
I see. Okay as long as the text is left aligned 👍  looks like the column width is the same width as the Text widget, so alignment of the Text widget within the column won't change wrt to cross axis alignment (unless there is a wider widget than the text widget added to the column at some point).